frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx

```diff
@@ -84,13 +83,7 @@ const AlertItem: React.FC<AlertItemProps> = ({ alert }) => {
       Icon={getSeverityIcon(getAlertSeverity(alert))}
       timestamp={getAlertTime(alert)}
       message={getAlertDescription(alert) || getAlertMessage(alert) || getAlertSummary(alert)}
-      documentationLink={
-        alertName
-          ? `https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html-single/troubleshooting_openshift_data_foundation/index#${toLower(
-              alertName,
-            )}_rhodf`
-          : null
-      }
+      documentationLink={alertName && alertDocumentationURL ? alertDocumentationURL : null}
```

Comment:
will there be any situation when alert name is not present but alertDocumentationURL is ??
nits: just passing `alertDocumentationURL` should work as well...
```suggestion
      documentationLink={alertDocumentationURL}
```

Reply:
Makes sense, updated.